.github/actions/build-image-terraform/action.yml

```diff
@@ -42,6 +42,7 @@ runs:
     - uses: chainguard-dev/actions/setup-registry@main
       with:
         port: 5000
+        crane-version: tip  # TODO: switch to default latest-release after next ggcr release
```

Comment:
This needs `setup-go` 😞

Reply:
Up do go hnsndnjnhsbhs hbehanhzhnxhdnh see nzjxnjenzhjee see insane;bhzhhehshehaklskrjshhdnhjjshdbhendhzhnzhjdenshn  the nrjdjnrjrndjjsndnndntnnznsnjznhjdjdnnenrnjtnrnnrnennjemnshmjdjrnmdjtmfnnrnzhntmjtnhdnndnfnhnndndn4'nxntnnrnnnhendjrmjdnrnh tmrmjennrnendtj nndnmthrnktmdmmrntnnrmdjrmnfntnbfbbnnndnrnjdjjdntntnnfruy